src/libs/queries/extApi/simulator.ts

```diff
@@ -56,45 +56,70 @@ export type SimulatorReturn = {
   gains?: number;
 };
 
+export interface SimulatorDataNew {
+  date: number;
+  price: string;
+  sell: string;
+  buy: string;
+  portfolioOverHodlInPercent: string;
+  portfolioValueInQuote: string;
+  hodlValueInQuote: string;
+  baseBalance: string;
+  basePortion: string;
+  quoteBalance: string;
+  quotePortion: string;
+}
+
+interface SimulatorBoundsNew {
+  sellMin: string;
+  sellMax: string;
+  buyMin: string;
+  buyMax: string;
+}
+
+export interface SimulatorReturnNew {
+  data: Array<SimulatorDataNew>;
+  bounds: SimulatorBoundsNew;
+  roiInPercent: string;
+  gainsInQuote: string;
+}
+
 export type SimulatorAPIParams = Omit<
   SimulatorResultSearch,
-  'buyIsRange' | 'sellIsRange'
+  'buyIsRange' | 'sellIsRange' | 'type' | 'overlappingSpread'
 >;
 
-export const useGetSimulator = (params: SimulatorAPIParams) => {
+export const useGetSimulator = (search: SimulatorResultSearch) => {
   return useQuery<SimulatorReturn, Error>(
-    QueryKey.simulator(params),
+    QueryKey.simulator(search),
     async () => {
       try {
+        const { buyIsRange, sellIsRange, type, spread, ...params } = search;
         const res = await carbonApi.getSimulator(params);
 
+        // TODO cleanup schema
```

Comment:
If you don't do it in this PR, can you open an issue with reference the issue id in the TODO comment ?